src/library_manager/lib_manager.c

```diff
@@ -697,6 +697,11 @@ static int lib_manager_store_library(struct lib_manager_dma_ext *dma_ext,
 	uint32_t preload_size = man_desc->header.preload_page_count * PAGE_SZ;
 	int ret;
 
+	if (preload_size < MAN_MAX_SIZE_V1_8) {
```

Comment:
The MAX_SIZE is a bit odd (allocation fails unless you allocate maximum size), but I think this is actually the size of the manifest, so the condition makes sense.

Reply:
It's actually a bit confusing. This is the maximum size of the manifest and such space is always reserved regardless of the actual size.

Reply:
I agree with you both, lets add a comment here since it is a bit confusing. This will explain to everybody the usage.